src/skmultiflow/trees/nodes/inactive_learning_node_perceptron.py

```diff
@@ -81,9 +81,9 @@ def update_weights(self, X, y, learning_ratio, ht):
         """
         normalized_sample = ht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
-        normalized_target_value = ht.normalized_target_value(y)
+        normalize_target_value = ht.normalize_target_value(y)
```

Comment:
The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.